src/library_webgpu.js

```diff
@@ -2557,7 +2557,10 @@ var LibraryWebGPU = {
     {{{ makeSetValue('info', C_STRUCTS.WGPUAdapterInfo.deviceID, '0', 'i32') }}};
   },
 
+  wgpuAdapterGetProperties__deps: ['$warnOnce'],
   wgpuAdapterGetProperties: (adapterId, properties) => {
+    warnOnce('getProperties is deprecated, use getInfo instead');
```

Comment:
```suggestion
    warnOnce('wgpuAdapterGetProperties is deprecated, use wgpuAdapterGetInfo instead');
```

Reply:
FYI I was following convention of using short names as in https://github.com/emscripten-core/emscripten/blob/c0aa4038c73ed8f64007767638c2a1be1319d330/src/library_webgpu.js#L1987

Reply:
Ah, we're definitely not consistent, but better to aim for clearer messages anyway.
(In the case of getMappedRange at least the name is quite specific. "getProperties" sounds very generic even though only one such thing exists.)